bittensor/_keyfile/keyfile_impl.py

```diff
@@ -143,6 +146,17 @@ def ask_password_to_encrypt() -> str:
         valid = validate_password(password)
     return password
 
+def keyfile_data_is_encrypted_nacl( keyfile_data:bytes ) -> bool:
+    """ Returns true if the keyfile data is NaCl encrypted.
+        Args:
+            keyfile_data ( bytes, required ):
+                Bytes to validate
+        Returns:
+            is_nacl (bool):
+                True if data is ansible encrypted.
+    """
+    return keyfile_data[:len('$NACL')] == b'$NACL'
```

Comment:
does it make sense to create a variable with the '$NACL' value to not use it twice?